### DISCUSSION TITLE
Center Focus Custom Layout

### DISCUSSION THREAD

**Discussion by bryanculver** [ianyh/Amethyst#1697] [Show and tell]
I found my Four Corners layout hard to work with a large screen and having a window that was the sole focus, like writing.

The main window always takes the full height and middle half of the screen. The other windows take up the sides.

I felt others might find it useful: https://gist.github.com/bryanculver/4c181881cca81d9963f49a77dd5ac127#file-center-focus-js

**Comment by SoftBoil** [comment]
Thank you for sharing your layout!

I'm experimenting with custom layouts as well. While developing yours, were you able to debug the javascript? I tried console.log commands but I can't find the output.

**Comment by jorgehezd** [comment]
Thank you for sharing.

I don't have any ability to read or develop. 

But I understand that with this configuration I can center the automatically focused window, is that correct?
